.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!--
+    Thanks for submitting a Pull Request (PR) to this project.
+    Your contribution to this project is greatly appreciated!
+
+    Please prefix the PR title with the resource name,
+    e.g. 'ResourceName: My short description'.
+    If this is a breaking change, then also prefix the PR title
```

Comment:
Can we replace this section for this module only to say that breaking changes are not allowed?

Reply:
@johlju - are you able to address this one?

Reply:
I think this is already addressed, and commented on in a Reviewable comment.

Reply:
Ah I see - I didn't look through the reviewable comments because I didn't think it had been used in this PR. I'll check them.